packaging/installer/README.md

```diff
@@ -1,44 +1,40 @@
-import { OneLineInstallWget, OneLineInstallCurl } from '@site/src/components/OneLineInstall/'
-import { InstallRegexLink, InstallBoxRegexLink } from '@site/src/components/InstallRegexLink/'
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-
 # Netdata Agent Installation
 
 Netdata is very flexible and can be used to monitor all kinds of infrastructure. Read more about possible [Deployment guides](/docs/deployment-guides/README.md) to understand what better suites your needs.
 
 ## Install through Netdata Cloud
 
-Netdata is a free and open-source (FOSS) monitoring agent that collects thousands of hardware and software metrics from any physical or virtual system (we call them _nodes_). These metrics are organized in an easy-to-use and -navigate interface.
+The easiest way to install Netdata on your system is via Netdata Cloud, to do so:
 
-Netdata runs permanently on all your physical/virtual servers, containers, cloud deployments, and edge/IoT devices.
-It runs on Linux distributions (Ubuntu, Debian, CentOS, and more), container/microservice platforms (Kubernetes clusters, Docker), and many other operating systems (FreeBSD, macOS), with no `sudo` required.
+1. Sign up to <https://app.netdata.cloud/>.
+2. You will be presented with an empty space, and a prompt to "Connect Nodes" with the install command for each platform.
+3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it.
 
-To install Netdata in minutes on your platform:
+Upon installation completing successfully, you should be able to see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).
```

Comment:
Duplicate text

<img width="1298" alt="Screenshot 2024-05-27 at 14 45 23" src="https://github.com/netdata/netdata/assets/22274335/cf5bad35-d678-4065-99e0-c404560eaf89">

Duplicate text

<img width="1298" alt="Screenshot 2024-05-27 at 14 45 23" src="https://github.com/netdata/netdata/assets/22274335/cf5bad35-d678-4065-99e0-c404560eaf89">

Reply:
let me find the duplicate and see what to do about it

Reply:
wow, now I get it

Reply:
I must've pressed the revert button on something and added the previous wording back in

Reply:
> let me find the duplicate and see what to do about it

I even posted duplicate report of this text!